test/cpp/inference/api/full_pascalvoc_test_preprocess.py

```diff
@@ -47,7 +47,7 @@
 
 def preprocess(img):
     img_width, img_height = img.size
-    img = img.resize((RESIZE_W, RESIZE_H), Image.ANTIALIAS)
+    img = img.resize((RESIZE_W, RESIZE_H), Image.LANCZOS)
```

Comment:
To pass CI, change `ANTIALIAS` to `LANCZOS`.

> `ANTIALIAS` was removed in Pillow 10.0.0 (after being deprecated through many previous versions). Now you need to use `PIL.Image.LANCZOS` or `PIL.Image.Resampling.LANCZOS`.


Ref: https://stackoverflow.com/questions/76616042/attributeerror-module-pil-image-has-no-attribute-antialias